src/main/java/Event.java

```diff
@@ -0,0 +1,24 @@
+public class Event extends Task {
+
+    protected String on;
+
```

Comment:
Perhaps some of the variable names can be improved further. For example instead of on, can try something like dueOn in your Event class and instead of by, can try completeBy in your Deadline class